src/frontend/js/components/CourseGlimpse/CourseGlimpseFooter.tsx

```diff
@@ -1,9 +1,16 @@
-import { useIntl } from 'react-intl';
+import { defineMessages, useIntl } from 'react-intl';
 
 import { Icon } from 'components/Icon';
 import { CommonDataProps } from 'types/commonDataProps';
 import { Course } from 'types/Course';
 
+const messages = defineMessages({
+  dateIconAlt: {
+    defaultMessage: 'Course date',
+    description: 'Course date logo alternative text for screen reader users',
```

Comment:
```suggestion
    defaultMessage: 'Course date',
    description: 'Course date logo alternative text for screen reader users',
    id: 'components.CourseGlimpseFooter.dateIconAlt'
```
You forget to add an `id` which is used to generate translations json files.